.github/workflows/auto_request_review.yml

```
@@ -0,0 +1,16 @@
+name: Auto Request Review
+
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```
As far as I read in your shared post, that change might make the trick to work with forks.
It's worth trying. Could you remove us as reviewers, make this change, and see how it behaves?